### ISSUE TITLE
Unable to run 'npm install' command, every time i am getting following issue. Spend around whole day but could not fix this. Please help me out on this.

### ISSUE THREAD

**Issue by shakeel2198** [npm/npm#19509]
npm ERR! path ../semver/bin/semver
npm ERR! code EPROTO
npm ERR! errno -71
npm ERR! syscall symlink
npm ERR! EPROTO: protocol error, symlink '../semver/bin/semver' -> '/home/vagrant/Code/samenfietsen/node_modules/gulp/node_modules/.bin/semver'

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2018-01-04T06_38_29_181Z-debug.log

**Comment by itsAnuga** [comment]
Running on Windows?
Try adding `–no-bin-links` when running `npm`.

**Comment by shakeel2198** [comment]
@Anuga Yes running on windows 10 but in vagrant. I setup homestead for Laravel.

**Comment by xMADOXX** [comment]
@shakeel2198 Hi man, this works for me : sudo npm i --no-bin-links

**Comment by madox35** [comment]
Exactly the same problem here, --no-bin-links doesn't work
